tests/core/daemon/test_daemon.py

```diff
@@ -382,7 +382,7 @@ async def num_connections():
     data = {"service": service_name}
     payload = create_payload("register_service", data, service_name, "daemon")
     await ws.send_str(payload)
-    message_queue = asyncio.Queue()
+    message_queue: asyncio.Queue = asyncio.Queue()
```

Comment:
Was this required for some reason?

Reply:
while working on the patch, mypy complained about not knowing the type of this variable. I'm not sure the thing that triggered it is still in the patch. sometimes mypy work in mysterious ways

Reply:
This test function isn't hinted and is also in the exclusion list so mypy wouldn't be looking inside anyways.  Maybe you temporarily hinted the function def at some point?

Reply:
mypy will enable additional hinting if you introduce hinting for added code. e.g. if you add a hinted parameter to a function where nothing is hinted, some type checking is enabled